packages/shared/schemas/transactionInitInput.ts

```diff
@@ -1,9 +1,33 @@
-import { z } from 'zod';
+import { z, ZodError, ZodIssue } from 'zod';
```

Comment:
sorry im new to this repo. What's Zod?

Reply:
I'm new to this too :) Overall it is used for input validation. It is a data/schema type declaration and validation library.